lib/tests/modules/declaration-positions.nix

```diff
@@ -27,23 +25,25 @@ assert builtins.unsafeGetAttrPos "a" (discardPositions { a = true; }) == null;
     }
   ];
 
-  options.nested.nestedLine30 = lib.mkOption {
-    type = lib.types.int;
-  };
+  options.nested.nestedLine30 = lib.mkOption { type = lib.types.int; };
```

Comment:
I like my `mkOption`s multi-line, consistently.
Will I have to adjust?

Reply:
Oh this is actually an inconsistency in the formatter that should be fixed, because
```nix
  options.nested.nestedLine30 = { type = lib.types.int; };
```

expands to
```nix
  options.nested.nestedLine30 = {
    type = lib.types.int;
  };
```

The same should be done here.

Reply:
This is actually less prevalent in practice than it looks like here, because as soon as you have at least two attribute values, it will also get expanded. And effectively all options in NixOS have both a `type` and a `description`, which makes two.

Reply:
I figured it might not be as bad.

Sometimes I start out with only a `description` or only a `type` if it's more "technical" like testing or trying something out.
I do anticipate the other attrs, which is why this stood out to me.